code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -21,6 +21,8 @@
 	family_heirlooms = list(/obj/item/toy/cattoy)
 	/// When false, this is a felinid created by mass-purrbation
 	var/original_felinid = TRUE
+	/// Brain size for scaling
+	var/brain_size = 0.8
```

Comment:
Why was this merged we’re trying to murder species vars, this could’ve easily been put on the brain???

Reply:
headcoder approved :shipit: